### PR TITLE
Emails stylized. PLEASE CHECK IT USING SENDGRID

### PR DESCRIPTION
**Pull request by jesuisnicolas**
I need someone to check if the messages are styled when sent using Sendgrid. My sendgrid account got temporarily suspended for some reason.

The mailer.html shouldn't have been uploded.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ node_modules
 mongo-volume
 tester.js
 .vscode
+mailer.html
```
Comment (MATTALUI) [inline .gitignore:7]: We don't want to ignore this? Otherwise we won't be able to update it. Is this right?

File: src/utils/mailer.js [hunk 2]
```diff
@@ -55,10 +57,11 @@ const buildNewUserEmailOptions = ({
...
     text: `A Litzen account has been associated with this email address. To confirm your account and start receiving announcements visit the following address: ${validationUrl}/`,
-    html: `
-      <p>A Litzen account has been associated with this email address. To confirm your account and start receiving announcements click <a href="${validationUrl}">here</a>.</p>
-      <p>If the above link does not work you can visit ${validationUrl}</p>
-      `,
+    html: mailCreator.validationEmail(validationUrl),
+    
+    // `<p>A Litzen account has been associated with this email address. To confirm your account and start receiving announcements click <a href="${validationUrl}">here</a>.</p>
+    //   <p>If the above link does not work you can visit ${validationUrl}</p>
+    //   `,
```
Comment (MATTALUI) [inline src/utils/mailer.js:64]: Once we've gotten this sorted we should cut these comments.

File: .gitignore (hunk 1 above)
Comment (jesuisnicolas) [reply .gitignore:7]: We don't need that file. I just used it to check the layout.

File: src/utils/mailer.js (hunk 2 above)
Comment (jesuisnicolas) [reply src/utils/mailer.js:64]: I'm on it.

File: .gitignore (hunk 1 above)
Comment (MATTALUI) [reply .gitignore:7]: That makes sense. We should remove it from the PR then.

**Review by MATTALUI — APPROVED** [review]
Ideally we'll trim anything we don't need here. No reason to stop it though.

**Pull request merged**